README.md

```diff
@@ -1,5 +1,12 @@
+[![Version](https://img.shields.io/github/v/release/kubescape/regolibrary)](releases)
+[![release-date](https://img.shields.io/github/release-date/kubescape/regolibrary)](releases)
+[![GitHub](https://img.shields.io/github/license/kubescape/kubescape)](https://github.com/kubescape/kubescape/blob/master/LICENSE)
+
 # Welcome to Kubescape Regolibrary!
 
+Kubescape Regolibrary is part of the Kubescape open-source Kubernetes security platform. It provides good practice software on security configurations for unmanaged Kubernetes clusters where you typically manage the Kubernetes cluster control plane and nodes.
+
```

Comment:
"It provides good practice software on security configurations for unmanaged Kubernetes clusters where you typically manage the Kubernetes cluster control plane and nodes." --> This sentence isn't very clear to me

@craigbox - do you have any suggestions here?

Reply:
I've used something like this

"This repository contains a library of security controls that codify Kubernetes best practices derived from the most prevalent security frameworks in the industry. [Kubescape] uses these controls to scan again running clusters or manifest files under development. They’re written in Rego, the purpose-built declarative policy language that supports Open Policy Agent (OPA)."

(In Q2 we should be doing a full docs rewrite of READMEs and moving examples out of them too.)

Reply:
@craigbox - Sure. let's sync regarding the README files new templates. It will be good for us to be familiar with it so we can start prepare it ahead